### ISSUE TITLE
ITVR - rebates that failed to post over to NCDA

### ISSUE THREAD

**Issue by kcabhar** [bcgov/itvr#417]
**Description:**
A rebate will fail to POST over to ncda if the dl# associated with the rebate is already in the NCDA's system. In fact, this POST will always fail.

So, since we send over rebates to ncda in batches of 100, and what goes in a batch is simply the "top" 100 rebates in our system without an ncda id (successfully posted rebates will be updated to have an ncda id), the failing POST mentioned above will be part of every batch, thus reducing our throughput.

This may become a problem as time goes on; we need to think about how to deal with this. We don't want to simply attempt to post each rebate only once; for example, if the ncda's server is down, we obviously want to re-attempt.

**Comment by kcabhar** [comment]
Comments from Jira:

**James - 31/Aug/22**
Not a priority - Tim to re-draft card.
